xdsl/transforms/experimental/hls_convert_stencil_to_ll_mlir.py

```diff
@@ -1208,24 +1208,14 @@ def match_and_rewrite(self, op: scf.ParallelOp, rewriter: PatternRewriter, /):
 class QualifyInterfacesPass(RewritePattern):
     module: builtin.ModuleOp
     declared_coeff_func: bool = False
+    called_coeff_func: bool = False
```

Comment:
This is read but never set anywhere? Is there some use of it missing?

Reply:
Ah yes, good catch